src/media/css/buttons.styl

```diff
@@ -130,6 +130,7 @@ a.button {
     }
 }
 
+.mkt-website-button,
```

Comment:
It appears as though I added this class and then changed my mind. This it the only reference to it.